changelog/_unreleased/2025-07-14-restore-kernel-handle-resetinterface.md

```diff
@@ -0,0 +1,8 @@
+### Changed
```

Comment:
this changelog does not match the schema
https://github.com/shopware/shopware/blob/trunk/changelog/_example.md

you can run also `bin/console changelog:check` locally to check it